fastlane/Fastfile

```diff
@@ -0,0 +1,40 @@
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+skip_docs
+
+keychain_path = Pathname.new(File.join(Dir.pwd, "..", "temp/temporary.keychain")).cleanpath.to_s
+
+lane :import_certificates do
+    # In theory, this will create the keys and download them if necessary.
+    # Failing because it somehow maps a Developer ID Application key to AppStore by mistake.
+    # We do this in a separate lane to ensure it doesn't taint our build (which is working fine, thank you very much).
+    sync_code_signing(
+        storage_mode: "git",
+        git_url: ENV['CERTIFICATE_REPOSITORY'],
+        git_basic_authorization: ENV['CERTIFICATE_REPOSITORY_AUTHORIZATION_KEY'],
+        type: "developer_id",
+        app_identifier: "uk.co.inseven.fileaway",
+        readonly: true,
+        skip_provisioning_profiles: true,
+        keychain_name: keychain_path,
+        keychain_password: ENV['TEMPORARY_KEYCHAIN_PASSWORD'],
+    )
+end
```

Comment:
This change should introduce documentation explaining how to import a Mac development certificate into the match certificate store given how broken it is.

Reply:
Done.